### ISSUE TITLE
Default configuration overrides desktop-dirname

### ISSUE THREAD

**Issue by bgamari** [syl20bnr/spacemacs#6741]
It seems that the `spacemacs` layer overrides `desktop-dirname` (in `spacemacs/init-desktop`) in such a way that the users own preference is ignored. This breaks the workflow where the user maintains a persistent desktop file in each of their project working directories as emacs insists on loading the project file from `~/.emacs.d/.cache`.

**Comment by bmag** [comment]
Where do you set `desktop-dirname`? Please explain more about your setup.

**Comment by bgamari** [comment]
@bmag I've tried simply using `customize-variable`, as well as adding
```lang=elisp
  (with-eval-after-load 'desktop
    (setq desktop-dirname ".")))
```
to `dotspacemacs/user-config`.

**Comment by TheBB** [comment]
I tried the same code, then

1. Start Emacs
2. `M-: (require 'desktop)`
3. `SPC hdv desktop-dirname`

:point_right: its value is `"."`

Do you observe differently?

**Comment by bgamari** [comment]
> :point_right: its value is `"."`

I reproduce what you observed.

However, I've investigated a bit further and it seems that there are a few confounding factors here,

* In addition to `desktop-dirname` there is also `desktop-path`. `desktop-dirname` doesn't appear at all in `customize-variables` which leads me to believe that Spacemacs shouldn't be touching it at all.
* I have `desktop-save-mode` enabled which means that `desktop.el` is loaded rather early during startup. It would appear that it's actually being loaded while `desktop-dirname` or `desktop-path` is set to `~/.emacs.d/.cache` since it immediately loads any desktop file present in that directory on startup.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
